tests/gds_kernel_latency.c

```diff
@@ -138,27 +138,28 @@ struct pingpong_context {
         int                     rxtot_size;
         size_t                  alloc_size;
         char                    *rx_flag;
-        int                      buf_size;
-        int                      buf_sizeexp;
-        int                      calc_size;
-        int                      rx_depth;
-        int                      pending;
-        struct ibv_port_attr     portinfo;
-        int                      gpu_id;
-        int                      kernel_duration;
-        int                      peersync;
-        int                      peersync_gpu_cq;
-        int                      consume_rx_cqe;
-        int                      gpumem;
-        int                      use_desc_apis;
-        int                      n_tx_ev;
-        int                      n_rx_ev;
-        int                      scnt;
-        int                      rcnt;
-        int                      skip_kernel_launch;
-        int                      exp_send_info;
-        int                      validate;
-        char                     *validate_buf;
+        int                     buf_size;
+        int                     buf_align;
```

Comment:
I think this var name is incorrect, it suggests alignment requirements, but instead it is an aligned size.